src/lib/game-saving/account.ts

```diff
@@ -233,7 +233,7 @@ export const makeOrUpdateSession = async (cookies: AstroCookies, userId: string,
 		path: '/',
 		maxAge: 60 * 60 * 24 * 365,
 		httpOnly: true,
-		sameSite: 'strict'
+		sameSite: 'lax'
```

Comment:
What does this change?

Reply:
By changing from "Strict" to "Lax" we make sure the session token is readable for "new user" so they don't face the "No Active Session" error during publishing process.

while still keeping things secure, "strict" blocks any session and only allowing the main site to do stuff, meanwhile "lax" as same security like strict so it still is prevent from CSRF attack but also allow it to share the session to popup